dotnet/sample/AutoGen.BasicSamples/CodeSnippet/TypeSafeFunctionCallCodeSnippet.cs

```diff
@@ -11,6 +12,15 @@
 #region weather_report
 public partial class TypeSafeFunctionCall
 {
+    private class GetWeatherSchema
```

Comment:
Is this change necessary?

Reply:
I made this change for one of the unit test, `AnthropicAgentFunctionCallMessageTest()` . Do you prefer if i create my own FunctionCall in `Anthropic.Tests` instead?

Reply:
Yes I feel like it would be better. This code snippet is used in website so changing it will also change the code example in the tutorial as well.

Reply:
Yes. i reverted this usage and added this example under Anthropic.Test proj instead